tools/driver/Driver.cpp

```diff
@@ -662,100 +662,101 @@ int Driver::MainLoop() {
   bool handle_events = true;
   bool spawn_thread = false;
 
-  if (m_option_data.m_repl) {
```

Comment:
The change here is just always taking the else branch, and instead checking if we're in REPL mode just before where we were previously calling m_debugger.RunCommandInterpreter(...) and calling m_debugger.RunREPL(...) instead. The rest is due to indentation change.

This was needed for --repl mode to still run the commands passed via -O (and written to commands_stream in the code above this line)